jaggeryapps/admin/templates/header.jag

```diff
@@ -53,13 +69,36 @@ function resolveUserName(user) {
     return user;
 }
 
+function getJsonFromFile(pathTofile) {
+    var cloudConfig = application.get(pathTofile);
+         if (cloudConfig==null) {
+             if(log.isDebugEnabled()){
+                log.debug("Reading Json file : " + pathTofile);
+         }
+
+        try{
+            var file = new File(pathTofile);
+            file.open("r");
+            var data = file.readAll();
+            cloudConfig = parse(data);
+            file.close();
+            application.put(pathTofile,cloudConfig);
+        } catch (e) {
+            log.error("Error while reading file " + pathTofile + "\n" + e);
```

Comment:
Can you check the exception handling logic here? Do we need to log and throw? If it is printed like this, will it print the stacktrace?